msi/release.sh

```diff
@@ -1,17 +1,56 @@
 #!/bin/sh
 
-# Build Git for Windows' .msi files
+me=$(basename $0)
+function showUsage() {
```

Comment:
I apologize. I changed this, but perhaps during interactive rebase this was re-introduced :disappointed: